### ISSUE TITLE
Option to allow replacing a sequence of characters by a line break should be documented

### ISSUE THREAD

**Issue by maettu-this** [notepad-plus-plus/notepad-plus-plus#11494]
### Description of the Issue
I have a string like this: `123\r\nABC\r\nXYZ` and would like NP++ replace `\r\n` by line breaks.

### Expected Behavior
 A simple option like "replace ... by line break" would allow me converting this string to:
```
123
ABC
XYZ
```

**Comment by alankilborn** [comment]
Replace in regular expression mode:

Find: `\\r\\n`
Replace: `\r\n`

This really isn't an "issue", it is a question.  Questions are better discussed here:  https://community.notepad-plus-plus.org/

**Comment by maettu-this** [comment]
Well, seeing this answer, yes it turned out to be a question. But not getting the logic of the advanced features of the `Replace` dialog, thus not finding the functionality, it was a feature request.

Also, what confuses me with the `Replace` dialog:

- Why is `Backward direction` greyed out when using `Regular expression`? `Wrap around` is active.
- `.` matching newline seems a very particular behavior when being used to regular expression.

Playing around I found that `Extended` mode also works.

I think there should be some explanation on these advanced features of the dialog. I found neither tooltips nor anything at https://npp-user-manual.org/.
